gateways/iap.sh

```diff
@@ -10,13 +10,22 @@
 # warmly appreciated.
 #----------------------------------------------------------------------------
 
+# prevent connection error 'Login profile size exceeds 32 KiB'
+# https://github.com/kyma-project/test-infra/issues/93#issuecomment-457263589
+account=`echo $(gcloud config list account --format "value(core.account)")`
+
+for key in $(gcloud compute os-login ssh-keys list | grep -v FINGERPRINT); do
+    echo "SSH key '$key' for account '$account' removed"
+    gcloud compute os-login ssh-keys remove --key $key
+done
+
 gw="$TUNNEL_GATEWAY_HOST"
 [ "X$TUNNEL_GATEWAY_USER" = X ] || gw="$TUNNEL_GATEWAY_USER@$TUNNEL_GATEWAY_HOST"
 
-$TUNNEL_GCLOUD_CMD compute ssh \
+$TUNNEL_SSH_CMD compute ssh -q \
```

Comment:
```suggestion
$TUNNEL_GCLOUD_CMD compute ssh -q \
```

Change this to `$TUNNEL_SSH_CMD` will make the command by default being `ssh -o StrictHostKeyChecking=no -o PasswordAuthentication=no compute ssh -q`..

Reply:
In case if module looks like this
```
module "db_tunnel" {
  source = "git::https://github.com/localdotcom/terraform-ssh-tunnel.git?ref=iap-tunnel"

  create             = true
  type               = "iap"
  ssh_cmd            = "gcloud"
  iap_gcp_project    = var.project
  iap_gcp_zone       = local.bastion_zone
  target_host        = module.mysql.instance_first_ip_address
  target_port        = 3306
  gateway_host       = local.bastion_hostname
  parent_wait_sleep  = "30"
  tunnel_check_sleep = "60"
}
```
the `ssh_cmd` is `gcloud`,  so it won't be overriden by `ssh -o StrictHostKeyChecking=no -o PasswordAuthentication=no compute ssh -q` and command will look like `gcloud compute ssh -q` in this case.

We also can change `ssh_cmd` in the module `db_tunnel` to `gcloud compute ssh -q` and then change `$TUNNEL_SSH_CMD compute ssh -q` to `$TUNNEL_SSH_CMD`